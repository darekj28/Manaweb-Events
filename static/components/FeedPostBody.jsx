class FeedPostBody extends React.Component {
	render() {
		return(
			<div className="FeedPostBody">
				{this.props.content}
			</div>
		)
	}
}