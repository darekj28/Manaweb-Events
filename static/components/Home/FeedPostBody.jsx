var React = require('react');
export default class FeedPostBody extends React.Component {
	render() {
		return(
			<div className="FeedPostBody">
				{this.props.content}
			</div>
		)
	}
}