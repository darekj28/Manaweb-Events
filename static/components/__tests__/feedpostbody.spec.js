import FeedPostBody from '../FeedPostBody.jsx';

describe('FeedPostBody', () => {
	it('renders correctly', () => {
		const fpb = renderer.create(
			<FeedPostBody content="a"/>
			);
		expect(fpb).toMatchSnapshot();
	});
});