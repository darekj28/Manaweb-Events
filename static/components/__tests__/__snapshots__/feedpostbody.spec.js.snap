exports[`FeedPostBody renders correctly 1`] = `
<div
  className="FeedPostBody">
  a
</div>
`;
